plugins/SitesManager/vue/src/SiteFields/SiteFields.vue

```diff
@@ -211,6 +232,9 @@ import {
   Setting,
   PasswordConfirmation,
 } from 'CorePluginsAdmin';
+import {
+  AnonymizeIp,
+} from 'PrivacyManager';
```

Comment:
This will need som fallbacks.

It is possible to deactivate the `PrivacyManager` plugin. Before it should just hide all the settings, now it breaks the site management completely. Something like `const AnonymizeIp = useExternalPluginComponent('PrivacyManager', 'AnonymizeIp');` and then try "AnonymizeIp !== null" to detect if the plugin is available.

And if not we should hide the privacy button 😬